cmd/osm-bootstrap/osm-bootstrap.go

```diff
@@ -490,7 +450,7 @@ func buildMeshRootCertificate(presetMeshRootCertificateConfigMap *corev1.ConfigM
 			APIVersion: "config.openservicemesh.io/configv1alpha2",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name: meshRootCertificateName,
+			Name: constants.DefaultMeshRootCertificateName,
```

Comment:
This isn't a change in behavior for this PR, this was a local constant (see line 52 above) and I moved it to the constant so it would be in one place to be used across tests as well.

Reply:
we can add a TODO at least?

Reply:
@steeling Could you explain more?

Reply:
Deleting isn't blocked but upon bootstrapper restart we will see issues.  I created https://github.com/openservicemesh/osm/issues/5237 to track. @ksubrmnn hopefully the issue helps explains and proposes a few options for solving.